### DISCUSSION TITLE
Is there a way to download a list of all manifests ids within a branch?

### DISCUSSION THREAD

**Discussion by DutchBaller** [SteamRE/DepotDownloader#580] [Q&A] [answered]
I want to get a list of manifests ids from a certain password protected branch but steamdb doesn't show them even if you have a password so I was wondering if this program has those capabilities or will have those features added.

**Comment by psychonic** [comment, marked as answer]
Steam only provides the one, current manifest for any given branch. You cannot get past ones without a time machine.
